### ISSUE TITLE
Okta widget focus styling not present for password input field

### ISSUE THREAD

**Issue by mark-vb-austin** [okta/okta-signin-widget#3651]
### Describe the bug

When the user selects/focuses on the password field within the Okta login widget  the focus styling should be applied similarly to the text input for Username/Email for A11Y

### Reproduction Steps

https://github.com/okta/okta-signin-widget/assets/7150628/9941c245-5865-4dee-9fd6-62a9bee1c6d5

### SDK Versions

We are using @7.1.1 but still present within the latest release 7.19.2

### Additional Information

I'm not sure if this was intentionally not added by the Okta team or a simple oversight.

If you look within the [accessibility.scss](https://github.com/okta/okta-signin-widget/blob/master/assets/sass/modules/_accessibility.scss) file, starting at line 12 you will see the box-shadow property is being applied to `input[type="text"]:focus` but there is no selector for `input[type="password"]:focus` which in turn would fix the issue.
